website/docs/cloud-docs/integrations/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+page_title: Integrations - HCP Terraform
+description: >-
+  This overview page lists HCP Terraform integrations with various third-party platforms and systems. 
+---
+
+# Overview
+
+The HCP Terraform ecosystem features a variety of integrations – applications
+designed to extend Terraform’s capabilities across various third-party systems
+and platforms. As demand grows for managing complex infrastructure across
+multiple clouds, data centers, and third-party services, our integrations are
+expanding.
+
+This section lists the current HCP Terraform integrations, which are API-based
+and leverage HCP Terraform's robust set of APIs. Each integration has its own
+dedicated page where you can find more details:
+
+1. [HCP Terraform Operator for Kubernetes](/terraform/cloud-docs/integrations/kubernetes)
+1. [ServiceNow Service Catalog for Terraform](/terraform/cloud-docs/integrations/service-now/service-catalog-terraform)
+1. [ServiceNow Service Graph Connector for Terraform](/terraform/cloud-docs/integrations/service-now/service-graph) 
+1. [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk)
+1. [Run Task integrations](/terraform/cloud-docs/integrations/run-tasks)
```

Comment:
I'd actually recommend pulling this out into a sentence underneath because it's custom and not scoped to a particular product i like the others in this list.
```suggestion
```